frontend/src/components/EnSavoirPlus.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function EnSavoirPlus() {
```

Comment:
Please rename your component to "LearnMore" for uniform language use

Reply:
Checked